purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -196,7 +196,12 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
         log(LogIntent.DEBUG, ConfigureStrings.APP_FOREGROUNDED)
         if (firstTimeInForeground || deviceCache.isCustomerInfoCacheStale(appUserID, appInBackground = false)) {
             log(LogIntent.DEBUG, CustomerInfoStrings.CUSTOMERINFO_STALE_UPDATING_FOREGROUND)
-            fetchAndCacheCustomerInfo(identityManager.currentAppUserID, appInBackground = false)
+            customerInfoRetriever.retrieveCustomerInfo(
+                identityManager.currentAppUserID,
+                fetchPolicy = CacheFetchPolicy.FETCH_CURRENT,
```

Comment:
With this policy, we will ignore the cache and just fetch from the backend, which is what we were doing before.